src/mscorlib/shared/System/Runtime/CompilerServices/RuntimeWrappedException.cs

```diff
@@ -2,41 +2,39 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-/*=============================================================================
-**
-**
-**
-** Purpose: The exception class uses to wrap all non-CLS compliant exceptions.
-**
-**
-=============================================================================*/
-
 using System;
 using System.Runtime.Serialization;
-using System.Diagnostics.Contracts;
 
 namespace System.Runtime.CompilerServices
 {
+    /// <summary>
+    /// Exception used to wrap all non-CLS compliant exceptions.
+    /// </summary>
     public sealed class RuntimeWrappedException : Exception
     {
+        private Object m_wrappedException; // EE expects this name
+
+#if CORECLR
         private RuntimeWrappedException(Object thrownObject)
```

Comment:
Nit: All existing places have the ifdef around the modifier only, e.g.: https://github.com/dotnet/coreclr/blob/26fbb1933b1ead4168f271ab4ced0fd9906adbe0/src/mscorlib/shared/System/Collections/ListDictionaryInternal.cs#L24